src/platforms/common/usage/sdk-fingerprinting.mdx

```diff
@@ -2,7 +2,7 @@
 title: "SDK Fingerprinting"
 sidebar_order: 30
 redirect_from:
-  - ../../../enriching-events/context/fingerprint-override/
+  - ../../../usage/events/enriching-events/context/fingerprint-override/
```

Comment:
I think this change was a mistake, although it looks like even the existing redirect here doesn't work

```suggestion
  - ../../../enriching-events/context/fingerprint-override/
```